pawno/include/PPC_PlayerCommands.inc

```diff
@@ -5145,3 +5248,52 @@ COMMAND:givelicense(playerid,params[])
 	// Let the server know that this was a valid command
 	return 1;
 }
+//This command let the admins to send an announce message
+COMMAND:announce(playerid, params[])
```

Comment:
This command needs a sscanf query to check if a text has been entered and to check that the text is not too long. Also it would be nice to use a custom style for the gametext.